{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/factories.py

```diff
@@ -12,7 +12,7 @@ class UserFactory(DjangoModelFactory):
 
     @post_generation
     def password(self, create: bool, extracted: Sequence[Any], **kwargs):
-        password = Faker(
+        password = extracted if extracted else Faker(
```

Comment:
The build is failing because this breaks black formatting. It's not clear from the output, but the right formatting is:

```python
        password = (
            extracted
            if extracted
            else Faker(
                "password",
                length=42,
                special_chars=True,
                digits=True,
                upper_case=True,
                lower_case=True,
            ).generate(extra_kwargs={})
        )
```

Would you mind updating this?

Reply:
@browniebroke I think you just did it, right?

Reply:
Yes, realised you gave me access to fix the branch. Thanks for that 🎉

Reply:
Many thanks to you guys for creating and maintaining this wonderful project, which greatly reduced my effort to set up my new project.